tools/db-checksum/db-checksum-config.yaml

```diff
@@ -0,0 +1,8 @@
+tablesOutputLocation: dboutputhashes
+getAllTables: true
+endBlockHeight: 5
+sqliteFile: ../pact-v1-chain-8.sqlite
```

Comment:
This is sqlite file I have on disk. I'll change it some reasonable default value before the close of this PR.